### PR TITLE
Feature/getValidatorStatus

### PR DESCRIPTION
**Pull request by AnnRiera**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/blockcoders/near-rpc-providers/blob/main/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/near-rpc-provider.ts [hunk 1]
```diff
@@ -372,4 +373,17 @@ export class NearRpcProvider extends JsonRpcProvider {
       })
     }
   }
+
+  async getValidatorStatus(blockHashOrBlockNumber: string[] | number[] | null[]): Promise<GetValidatorStatusResponse> {
```
Comment (bzuker) [inline src/near-rpc-provider.ts:377]: I think maybe a better API for the end user is to provide a string or a number (or nothing) and we can then wrap those values in an array ourselves. 
So the type for this method would be `async getValidatorStatus(blockHashOrBlockNumber?: string | number)`
And then when when doing the `send`, we can wrap the block in an array, something like: `send('validators', [blockHashOrBlockNumber || null])`

This way the user can use it by doing `provider.getValidatorStatus('my-validator')`

**Review by bzuker — DISMISSED** [review]
Added a possible improvement, afterwards LGTM!

File: src/near-rpc-provider.ts (hunk 1 above)
Comment (AnnRiera) [reply src/near-rpc-provider.ts:377]: Ohh, nice! I'll fix this right now!

**Review by bzuker — APPROVED** [review]
LGTM

**Pull request merged**